packages/cli/src/ui/components/messages/InfoMessage.tsx

```diff
@@ -24,7 +24,7 @@ export const InfoMessage: React.FC<InfoMessageProps> = ({ text }) => {
       </Box>
       <Box flexGrow={1}>
         <Text wrap="wrap" color={theme.status.warning}>
-          <RenderInline text={text} />
+          <RenderInline text={text} defaultColor={theme.status.warning} />
```

Comment:
remove the theme.status.warning color on the line before as it is redundant. I suspect the entire outer Text object might be redundant but you should double check.